### PR TITLE
Stop clusters

### PR DESCRIPTION
**Pull request by yeesian**
I am quite pleased with the reflected changes in `test/multiroute-construction.jl`; you have identified a good set of cases!

### CODE REVIEW

File: test/multiroute-construction.jl [hunk 1]
```diff
@@ -141,7 +141,7 @@ module MultiRoutesMainNetwork
         @test [stns[u] for u in np.transferstop[harvard, ashmont][1]] == ["place-harsq","place-jfk"]
         @test [stns[u] for u in np.transferstop[harvard, braintree][1]] == ["place-harsq","place-jfk"]
         @test isassigned(np.stage, ashmont, braintree) == false # can't go from one branch to another
-        @test isassigned(np.stage, longwood, symphony) == false # can't go from one branch to another
+        @test isassigned(np.stage, longwood, symphony) == true # can go from one branch to another
```
Comment (juliayyan) [inline test/multiroute-construction.jl]: Why is it that now it's possible to go from one branch of the green line to another?

Comment (juliayyan) [reply test/multiroute-construction.jl]: Ah I see, it's because they take the green line to the 1 bus instead of the green line to the green line.

**Comment by yeesian** [comment]
That's a good question. I was just updating the tests to work, and will
look into it after lunch

File: test/multiroute-construction.jl (hunk 1 above)
Comment (yeesian) [reply test/multiroute-construction.jl]: thanks for looking into this! I'll update the comment

**Comment by yeesian** [comment]
I've removed `travis/pr` from the automated tests in the travis settings since it's just a repeat of `travis/push`.

That aside, this PR is now ready for review: here's [the changelog](https://github.com/juliayyan/TransitNetworks.jl/pull/11/files/21e085f48ebcee947518d075deafb4933468adc6..5424d12062e727b2cc8b16b5446d89982573483d) since your last comment. It's mainly me removing/commenting-out unused methods in `src/networkmethods.jl` since we've made changes to the network object in the last PR.

**Comment by juliayyan** [comment]
thanks! It looks good.  Is the idea that these commented-out methods would stay in SchedulingModels since they're pretty specific to those models?

**Comment by yeesian** [comment]
> Is the idea that these commented-out methods would stay in SchedulingModels since they're pretty specific to those models?

Yeah, they could be useful at some point in this package. But until then, I agree it might be better to have it implemented closer to where it'll be used.

**Pull request merged**